tcms/testruns/templates/testruns/search.html

```diff
@@ -105,7 +105,7 @@
             </thead>
         </table>
 
-        {% include 'pagination.html' %}
+        {% include 'include/pagination.html' %}
```

Comment:
include references are updated but the file itself has not been moved (see `git mv`). That's why you get multiple failures.  This also means you haven't tried to load the application locally in the browser and check any of the pages you modify.  Next time test locally.

Reply:
actually, I suppose that locally it works for him, because the file is there, he just hasn't added it to the commit.